package.json

```
@@ -26,7 +26,7 @@
   },
   "scripts": {
     "test": "npm run test:code && npm run test:tsc && npm run test:types",
-    "test:code": "c8 --check-coverage --lines 80 --functions 80 --branches 80 --statements 80 ava test/*.test.js",
+    "test:code": "c8 --check-coverage --lines 100 --functions 100 --branches 100 --statements 100 ava test/*.test.js",
```

There is a sweet shortcut for that 😁 

```suggestion
    "test:code": "c8 --100 ava test/*.test.js",
```